src/components/ColorPicker.tsx

```diff
@@ -369,6 +369,16 @@ export const ColorPicker = ({
 }) => {
   const pickerButton = React.useRef<HTMLButtonElement>(null);
   const coords = pickerButton.current?.getBoundingClientRect();
+  const mainmenuContentCoords = document
+    .querySelector(".mainmenu-content")
+    ?.getBoundingClientRect();
```

Comment:
`document.querySelector()` isn't gonna work with multi-instance scenarios

Reply:
camel case -> `mainMenuContentCoords` :)
```suggestion
  const mainMenuContentCoords = document
    .querySelector(".main-menu-content")
    ?.getBoundingClientRect();
```